spec/controllers/branches_controller_spec.rb

```diff
@@ -129,6 +129,23 @@
       end
     end
 
+    context "with extra convergence branche and one non-convergance" do
+      let!(:non_master_branch) { FactoryGirl.create(:branch, :name => 'feature-branch', convergence: false, repository: repository) }
+      let!(:convergence_branch) { FactoryGirl.create(:branch, :name => 'convergence', convergence: true, repository: repository) }
```

Comment:
style nit: since these variables are never referenced you should create them in a `before` loop.